internal/agent/connect_service.go

```diff
@@ -50,7 +54,15 @@ func (ic *identityConnectService) Connect() entity.Identity {
 
 		logger.WithField(config.TracesFieldName, config.FeatureTrace).Tracef("connect request with fingerprint: %+v", f)
 
-		ids, retry, err := ic.client.Connect(f)
+		metatada, err := ic.metadataHarvester.Harvest()
+		if err != nil {
+			logger.WithError(err).Error("metadata harvest failed")
+			time.Sleep(1 * time.Second)
```

Comment:
Why is it waiting 1 second? I see it's also done for the fingerprintHarvest, should we add a small comment?

Reply:
We commented about it in the standup so I close this comment